.github/workflows/black.yml

```diff
@@ -31,5 +31,5 @@ jobs:
 
     - name: Checking the code for formatting issues
       run: |
-          black --check --verbose $(git ls-files './tests/rest/test_*.py') $(git ls-files './samples/rest/bing*.py')
```

Comment:
nit: You could even consider splitting this PR in 2 separate ones to reduce the number of modified files. 

It doesn't matter so much